src/components/PlacesAutocomplete.vue

```diff
@@ -186,25 +186,28 @@ export default {
     }
   },
   methods: {
-    fetchPlaces (query, update, abort) {
+    async fetchPlaces (query, update, abort) {
       if (query.length < 2) {
         abort()
         return
       }
 
-      update(async () => {
-        this.showLoadingSpinner()
+      this.showLoadingSpinner()
 
-        if (!query) {
-          this.places = []
-        } else {
-          let places = await search(query, { country: this.searchCountry })
-          places = places.slice(0, 5)
+      // put async operations outside `update` function
+      // otherwise the select menu doesn't display correctly
```

Comment:
Wondering when and why it broke.
Maybe an undetected regression after a Quasar update.

Reply:
We didn't see it broken because the geocoding request was "fast enough".
Empirically, the menu displays correctly if the response is returned below 40 milliseconds.